Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/core/IConceptTypesService.java

```diff
@@ -0,0 +1,14 @@
+package edu.asu.conceptpower.app.core;
+
+import edu.asu.conceptpower.app.core.impl.ConceptTypesService.ConceptTypes;
+
+public interface IConceptTypesService {
+
+    /**
+     * This method returns the concept type based on the id parameter.
+     * 
+     * @param id
+     * @return ConceptTypes
+     */
```

Comment:
javadoc should be a bit more specific. what are the rules that decide what id is what type?